main.py

```diff
@@ -1,15 +1,20 @@
 from sketchy import Sketchy
 from ball import Ball
+from vector import Vector
 
 class MyDrawing(Sketchy):
 
     def setup(self):
-        return
+        width, height = [600, 400]
```

Comment:
Are you planning on using `ball.py` and `main.py` to test every new feature you make? If so consider having different implementations for each test so you have the tests later when you are making additions or fixes to `vector.py`.